content/en/docs/reference/feature-gates.md

```diff
@@ -1,226 +0,0 @@
----
```

Comment:
Why was this content removed?

Reply:
It's moved, not removed. New location here in the build: https://deploy-preview-8624--kubernetes-io-master-staging.netlify.com/docs/reference/command-line-tools-reference/feature-gates/
or in the repo at /content/en/docs/reference/command-line-tools-reference/feature-gates.md
(to put it where it appeared in Jekyll builds, thanks to the TOC yaml files)